### PR TITLE
Move instrument fields to root

### PR DESCRIPTION
**Pull request by uittenbroekrobbert**
# Description

The fields with more information about the instrument have been moved to the root level instead of the category 'instrument'.

The json schema has been reformatted as well, which adds some unwanted noise to this PR, apologies for that.

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [X]  I have tested the changes locally and verified that they work as expected.
- [X]  I have followed the project's coding conventions and style guidelines.
- [X]  I have rebased my branch onto the latest commit of the main branch.
- [X]  I have squashed or reorganized my commits into logical units.
- [X]  I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

File: instruments/iama.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 systemcard_path: .assessments[]               # append iama.yaml to 'assessments' list in system card
 schema_version: 1.1.0
 
-instrument:
+metadata:
```
Comment (berrydenhartog) [inline instruments/iama.yaml:4]: i would remove metadata and move all fields inside it to the root

**Review by berrydenhartog — COMMENTED** [review]
I would move all fields inside

**Review by anneschuth — APPROVED** [review]
@berrydenhartog will this break your current PR?

**Review by berrydenhartog — COMMENTED** [review]
LGTM

**Review by berrydenhartog — APPROVED** [review]
LGTM

**Pull request merged**